omnibus/config/projects/chef-fips.rb

```diff
@@ -36,7 +36,7 @@
 
 # Global FIPS override flag.
 override :fips, enabled: true
-override :'ruby-windows', version: "2.0.0-p647"
+override :ruby, version: "2.1.7"
```

Comment:
The source for it is :) - we've had in our cache from non-windows builds for a long time.